src/components/ColorPicker/ColorResult.vue

```diff
@@ -87,15 +87,13 @@ export default {
       width: 100%;
       border-radius: 0.5em;
       padding: 16px;
-      margin: 8px 8px 16px 8px;
+      margin: 8px 8px 16px;
       color: var(--darker-color);
       font-size: 14px;
     }
 
     .video-color--result-item {
-      margin-bottom: 12px;
-      margin-right: 8px;
-      margin-left: 8px;
+      margin: 0 8px 12px;
```

Comment:
compared to this:
```css
margin-bottom: 12px;
margin-right: 8px;
margin-left: 8px;
```
can be shorthand by writing this `margin: 0 8px 12px;`

reference: https://web.dev/learn/css/spacing/#margin

Reply:
Nice catch, thanks